AutoGrade/models.py

```diff
@@ -171,12 +174,13 @@ class Submission(models.Model):
     passed          = models.IntegerField(default=0)
     failed          = models.IntegerField(default=0)
     publish_date    = models.DateTimeField('date published', default=datetime.now)
+    late_sub       = models.BooleanField(default=False)
 
-    def get_score(self):
+    def get_score(self, late_days):
         total = self.passed + self.failed
         if total == 0:
             return 0
-        return float(self.passed) * self.assignment.total_points / total
+        return (float(self.passed) * self.assignment.total_points / total)- late_days*5
```

Comment:
Score is not affected by late submission. They are only allowed `N` late days per course but those are not penalized. (See my original description on #9)